daemon/container_operations_unix.go

```diff
@@ -129,10 +129,10 @@ func (daemon *Daemon) DisconnectFromNetwork(container *container.Container, netw
 		if container.RemovalInProgress || container.Dead {
 			return errRemovalContainer(container.ID)
 		}
-		if _, ok := container.NetworkSettings.Networks[networkName]; !ok {
-			return fmt.Errorf("container %s is not connected to the network %s", container.ID, networkName)
+		if _, ok := container.NetworkSettings.Networks[n.Name()]; !ok {
```

Comment:
This will fail in the case where the Network (`n`) is nil. This is a recent fix to support a case of disconnecting a container from a network even if the network is not in the DB.